introduction.md

```diff
@@ -0,0 +1 @@
+I'm not going to tell you anything about myself. This is just a test file.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.